command/maintenance_window_delete.go

```diff
@@ -23,5 +32,27 @@ func (c *MaintenanceWindowDelete) Synopsis() string {
 }
 
 func (c *MaintenanceWindowDelete) Run(args []string) int {
+	var mwID string
+	flags := c.Meta.FlagSet("maintenance-window delete")
+	flags.Usage = func() { fmt.Println(c.Help())}
```

Comment:
Based on how this line is formatted, it seems like `go fmt` wasn't ran on save. Could you please run `go fmt` on the files changed in this PR?